integration-tests/features/dependency_editor_analyses.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Dependency Editor Stack-analyses API behaviour
+
+  Scenario: Check that the API entry point
+    Given System is running
+    When I access /api/v1/depeditor-analyses
```

Comment:
What is the functional difference between the first 2 scenarios?

Reply:
These are both GET requests that do not include the required token, correct?

Reply:
The test that receives the 401 return code should include an invalid auth token, yes?

Reply:
first scenario is 
    header = no/invalid access token
    pyaload = None
2nd is
   header = valid access token
   payload = None

Reply:
depeditor-analyses is POST end point.